tests/benchmark_msbar_evolution.py

```diff
@@ -2,7 +2,7 @@
 """This module benchmarks MSbar mass evolution against APFEL."""
 import numpy as np
 
-from eko.msbar_masses import evolve_msbar_mass
+from eko.msbar_masses import compute_msbar_mass, evolve_msbar_mass, compute_msbar_mass
```

Comment:
You're importing twice `compute_msbar_mass`.

Moreover, even here I'd simplify names and use the scope instead
```py
from eko import msbar_masses

msbar_masses.compute
msbar_masses.evolve
```
When possible I'd always use syntax to convey semantics, it's more meaningful and explicit (and from my personal point of view more beautiful, since you repeat less).

Reply:
True, this is a good point, I'll change it.